extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/README.md

```diff
@@ -90,10 +90,13 @@ You can connect this channel to a Rollershutter or Dimmer item.
 
 ### Channel Type "contact", "switch"
 
-* __on__: A number (like 1, 10) or a string (like "ON"/"Open") that is recognised as on state.
-* __off__: A number (like 0, -10) or a string (like "OFF"/"Close") that is recognised as off state.
+* __on__: A number (like 1, 10) or a string (like "ON"/"Open") that is recognised as on/open state.
```

Comment:
okay, here you combine the description of switch and contact... Other technical reasons? Otherwise existing channel configurations will be lost. This (the change in config description definition) is API breaking, isn't it?